acoustid/future/fpindex/client.py

```
@@ -0,0 +1,282 @@
+
+
+class FingerprintIndexClientError(Exception):
+    """Base exception for fingerprint index client errors"""
+
+    pass
+
+
+class FingerprintIndexClient:
+    """Asyncio client for the acoustid-index HTTP API"""
+
+    def __init__(self, base_url: str, timeout: float = 30.0):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)